### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by jonas-45**
In this milestone, Forms were added to enable users to add posts and comments
A like post button is added to allow users to like posts
To achieve the above, a number of controllers and the views for the corresponding actions were also created. The actions that were considered in this milestone are;

- [ ] new
- [ ] create

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
```
Comment (brytebee) [inline app/controllers/likes_controller.rb:1]: The LikesController should also contain a `create` action. Some of the logic here should be separated into the create action.

This is to ensure that a users can like posts.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @jonas-45 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    @like = @post.likes.new
+    if @like.save
+      flash[:notice] = 'Like added successfully!'
+      redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
+    else
+      flash[:alert] = "Couldn't add a like!"
+      puts @like.errors.full_messages
+      render :new, status: :unprocessable_entity
+    end
+  end
```
Comment (laiifuu) [inline app/controllers/likes_controller.rb]: Nice job adding the create function to the likes controller, however, when I tried adding likes, it didn't work. I even checked the rails console to see the likes records that are saved, and it returned an empty list:
image
Kindly fix this issue.

**Review by laiifuu — CHANGES_REQUESTED** [review]
## Hi @jonas-45 :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

cat-driving-serious

 ## Highlights
- Green linters check :heavy_check_mark: 
- Some of the required changes were implemented :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb (hunk 2 above)
Comment (jonas-45) [reply app/controllers/likes_controller.rb]: okay on it. thanks

**Review by sumairq — APPROVED** [review]
### Hi @jonas-45 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Forms functioning properly
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**